ruby/basic_ruby/nested_collections.md

```diff
@@ -328,7 +328,9 @@ Amazing! We have found a great solution to returning an array that only contains
 </div>
 
 ### Additional Resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental.
+This section doesn't contains helpful links to other content yet! 
+Any suggestions ?
+Contribute on Github!
```

Comment:
We use specific wording for cases like this. It is outlined in our [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md).

The Additional Resources section should look like:

```suggestion

This section contains helpful links to related content. It isn’t required, so consider it supplemental.

*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```